app/assets/javascripts/student_profile_v2/profile_details.js

```diff
@@ -3,14 +3,46 @@
   var dom = window.shared.ReactHelpers.dom;
   var createEl = window.shared.ReactHelpers.createEl;
   var merge = window.shared.ReactHelpers.merge;
-
+  var FeedHelpers = window.shared.FeedHelpers;
   var QuadConverter = window.shared.QuadConverter;
 
+  var Colors = {
+    math: '#e8e9fc',
+    reading: '#ffe7d6',
+    procedural: '#e8fce8'
+  };
+  var styles = {
+    feedCard: {
+      marginBottom: '10px',
+      borderRadius: '10px',
+      fontSize: '10pt'
+    },
+    feedCardHeader: {
+      fontSize: '120%',
```

Comment:
Can you try sticking with absolute pixel values for these sizes?  The rest of the app isn't responsive for font size (and doesn't need to be since we aren't optimizing this for mobile use), so expressing this as a percent doesn't really add more over pixels.